src/index.js

```diff
@@ -432,19 +605,23 @@ async function handleSendTransaction(engine, payload, end) {
 						} else if(response && response.result) {
 							let data = {};
 							data.signature = response.result;
-							data.signer = account;
-							data.message = engine.messageToSign;
-							data.messageLength = messageLength;
+							data.from = account;
+							data.to = payload.params[0].to.toLowerCase();
 							data.apiId = api.id;
 							data.dappId = engine.dappId;
-							data.params = paramArray;
-							if(payload.params[0].value) {
-								data.value = payload.params[0].value;
-							} else {
-								data.value = "0x0";
-							}
+
+							data.data = payload.params[0].data;
+							data.nonceBatchId = config.NONCE_BATCH_ID;
+							data.expiry = config.EXPIRY;
+							data.baseGas = config.BASE_GAS;
+							data.userContract = userContractWallet;
+							data.value = web3.utils.toHex(payload.params[0].value || 0);
 							data.gasPrice = gasPrice;
-							data.gasLimit = gasLimit;
+							data.gasLimit = gasLimit?gasLimit:0;;
```

Comment:
;; one extra semicolon is there.

Reply:
gasLimit 0 means it has not been set in request, so biconomy will set a gasLimit at backend.